app/shared/page.tsx

```diff
@@ -0,0 +1,73 @@
+"use client";
+
+import React, { useEffect, useState } from "react";
+import styles from "@/styles/FolderPage.module.css";
+import LinkCardComponent from "@/components/LinkCardComponent";
+import { useRequest } from "@/hooks/useRequest";
+import { SampleFolder } from "@/types";
+
+const SharedPage = () => {
+  const [folderInfo, setFolderInfo] = useState<SampleFolder>({});
+
+  const { data, request } = useRequest({
+    url: "api/sample/folder",
+    method: "GET",
+  });
+
+  useEffect(() => {
+    request();
+  }, []);
+
+  useEffect(() => {
+    if (data) {
+      setFolderInfo(data);
+    }
+  }, [data]);
+
+  return (
+    <>
+      <section className={styles.folder_main_section}>
+        <img
+          src={folderInfo?.folder?.owner?.profileImageSource ?? ""}
+          className={styles.folder_user_img}
+        />
+        <p className={styles.folder_user_name}>
+          {folderInfo?.folder?.owner?.name ?? ""}
+        </p>
+        <strong className={styles.folder_name}>
+          {folderInfo?.folder?.name ?? ""}
+        </strong>
+      </section>
+      <section className={styles.folder_info_section}>
+        <div className={styles.folder_search_input_box}>
+          <img src="/images/Search.svg" />
+          <input
+            className={styles.folder_search_input}
+            placeholder="링크를 검색해 보세요"
+          />
+        </div>
+
+        {folderInfo?.folder?.links ? (
+          <article className={styles.folder_card_body}>
+            {folderInfo?.folder?.links?.map((e, i) => {
+              return (
+                <LinkCardComponent
+                  key={e.url + i}
+                  imgSrc={e.imageSource}
+                  createdAt={e.createdAt}
+                  desc={e.description}
+                  url={e.url}
+                  type={"static"}
+                />
+              );
+            })}
+          </article>
+        ) : (
+          <></>
+        )}
```

Comment:
우선 삼항연산자에서 조건식이 false일때 표현해줄게 없다면, 이렇게 && 연산자를 써주는게 더 명확할 것 같아요!
```suggestion
        {folderInfo?.folder?.links && (
          <article className={styles.folder_card_body}>
            {folderInfo?.folder?.links?.map((e, i) => {
              return (
                <LinkCardComponent
                  key={e.url + i}
                  imgSrc={e.imageSource}
                  createdAt={e.createdAt}
                  desc={e.description}
                  url={e.url}
                  type={"static"}
                />
              );
            })}
          </article>
        )}
```